integration-tests/oidc-wiremock-providers/src/test/java/io/quarkus/it/oidc/providers/OidcWellKnownProvidersTest.java

```diff
@@ -0,0 +1,169 @@
+                                        + "24323.1f4c41812b286422cbce183a9f083fa58f7c2761c281c2be483a376694f56274&state"
+                                        + "={{request.query.state}}")
+                                .withBody("")
+                                .withTransformers("response-template")));
+
+        wireMockServer.stubFor(
+                get(urlMatching("/auth/slack/openid/connect/keys.*"))
+                        .willReturn(aResponse()
+                                .withHeader("Set-Cookie", "{{request.headers.Set-Cookie}}")
+                                .withHeader("Content-Type", "application/json")
+                                .withBody(
+                                        """
+                                                {
+                                                  "keys": [
+                                                    {
+                                                     "kid": "1",
+                                                     "kty":"RSA",
+                                                     "n":"iJw33l1eVAsGoRlSyo-FCimeOc-AaZbzQ2iESA3Nkuo3TFb1zIkmt0kzlnWVGt48dkaIl13Vdefh9hqw_r9yNF8xZqX1fp0PnCWc5M_TX_ht5fm9y0TpbiVmsjeRMWZn4jr3DsFouxQ9aBXUJiu26V0vd2vrECeeAreFT4mtoHY13D2WVeJvboc5mEJcp50JNhxRCJ5UkY8jR_wfUk2Tzz4-fAj5xQaBccXnqJMu_1C6MjoCEiB7G1d13bVPReIeAGRKVJIF6ogoCN8JbrOhc_48lT4uyjbgnd24beatuKWodmWYhactFobRGYo5551cgMe8BoxpVQ4to30cGA0qjQ",
+                                                     "e":"AQAB"
+                                                     }
+                                                  ]
+                                                }
+                                                """)));
+
+        wireMockServer.stubFor(
+                post(urlMatching("/auth/slack/api/openid\\.connect\\.token.*"))
+                        .willReturn(aResponse()
+                                .withHeader("Set-Cookie", "{{request.headers.Set-Cookie}}")
+                                .withHeader("Content-Type", "application/json")
+                                .withBody(
+                                        """
+                                                {
+                                                  "ok": true,
+                                                  "access_token": "xoxp-7925551513107-7925645662178-7911177365927-reduced",
+                                                  "token_type": "Bearer",
+                                                  "id_token": "%s",
+                                                  "state": ""
+                                                }
+                                                """.formatted(OidcWiremockTestResource.getIdToken("Rosetta",
+                                                "7925551513107.7922794171477", Map.of("name", "Rosetta"))))
+                                .withTransformers("response-template")));
+
+        wireMockServer.stubFor(
```

Comment:
Sorry, is Slack giving email in the ID token, if not, then please ignore my earlier comment about the UserInfo injection, otherwise we can drop the userinfo stub

Reply:
According to the API Slack ID token has email, therefore I'll drop UserInfo https://api.slack.com/methods/openid.connect.token.